### PR TITLE
Scenes: SceneApp example in core 

### PR DESCRIPTION
**Pull request by torkelo**
This revives the first scene app demo I did. 

* Makes it a nested route under scenes (and available from the scenes list page) 
* Makes the section nav work (was not working due to PluginPage expecting context to get section nav) 

Not really sure where this should live long term (or if we don't need it). But it's nice to have a SceneApp example in core. 

but it could potentially be moved to the demo / examples app that will be part of scenes repo after https://github.com/grafana/scenes/pull/10

### CODE REVIEW

**Comment by leventebalogh** [comment]
Maybe it could live (or we can add another example with some docs) in https://github.com/grafana/grafana-plugin-examples? What do you think @torkelo?

**Comment by torkelo** [comment]
@leventebalogh maybe, not sure. It could also be moved to https://github.com/grafana/scenes/tree/main/packages/scenes-app 

but not sure where is best yet, think it could be good to have this in core for a while to just have an example that uses SceneApp & SceneAppPage in core

**Comment by leventebalogh** [comment]
Oki, I see. Even if we have put these in core or in the grafana/scenes repo, I would probably lean towards having a simple example app for using scenes in the grafana/grafana-plugin-examples repo as well, as we are imagining it to be the entry point for people interested in what's possible to be done using plugins. Then that simple example could link for more detailed usage and documentation to the grafana/scenes repo maybe.

**Comment by torkelo** [comment]
@leventebalogh yea, I think your right. Could be good with a full blown example like this in that repo

**Comment by mckn** [comment]
> @leventebalogh yea, I think your right. Could be good with a full blown example like this in that repo

Another benefit with adding it in that repo is that we can cover the example with tests that will be run regularly against the grafana/grafana main branch to verify that it is functional moving forward.

**Comment by torkelo** [comment]
@mckn I think that could be good place, not sure yet. But for the next couple of months I think I prefer this in core or in the scens repo as we will need to continuously update and break the scenes APIs . When scenes is more stable I think we can move it somewhere else. Or it actually becomes a core feature :shrug:  (with some more love)

**Comment by mckn** [comment]
> @mckn I think that could be good place, not sure yet. But for the next couple of months I think I prefer this in core or in the scens repo as we will need to continuously update and break the scenes APIs . When scenes is more stable I think we can move it somewhere else. Or it actually becomes a core feature 🤷 (with some more love)

Awesome, I think that makes sense! Great work on this 🙌🏻

File: public/app/features/scenes/apps/scenes.ts [hunk 1]
```diff
@@ -0,0 +1,514 @@
...
+  const scene = new EmbeddedScene({
+    $variables: getVariablesDefinitions(),
+    $data: httpHandlerQueries,
+    $timeRange: new SceneTimeRange({ from: 'now-1h', to: 'now' }),
+    controls: [
+      new VariableValueSelectors({}),
+      searchBox,
+      new SceneControlsSpacer(),
+      sceneToggle,
+      new SceneTimePicker({ isOnCanvas: true }),
```
Comment (dprokop) [inline public/app/features/scenes/apps/scenes.ts:228]: Added Refresh pickers where they were useful.

File: public/app/routes/routes.tsx [hunk 2]
```diff
@@ -580,9 +580,10 @@ export function getDynamicDashboardRoutes(cfg = config): RouteDescriptor[] {
       ),
     },
     {
-      path: '/scenes/embedded/:name',
+      path: '/scenes/grafana-monitoring',
```
Comment (dprokop) [inline public/app/routes/routes.tsx:583]: 👍

**Review by dprokop — APPROVED** [review]
Works great!

@mckn @leventebalogh - I think at some point we can move https://github.com/grafana/scenes-app-template/  to examples repository.

**Pull request merged**